src/main/java/com/gildedrose/RedWine.java

```diff
@@ -0,0 +1,39 @@
+package com.gildedrose;
```

Comment:
Question: are we at a point where having a package name for items would make things tidy?

Reply:
We are planning to do this next PR!